scripts/generate_test_suites.py

```diff
@@ -0,0 +1,44 @@
+import os
+import sys
+
+parts = sys.argv[1]
+
+instrumentation_tests_path = 'instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests'
+
+
+def file_has_tests(path, file):
+	file_content = open(f'{path}/{file}').read()
+	return '@Test' in file_content
```

Comment:
In theory it's possible that the file doesn't have `@Test` annotation, but has tests. For example,
`class ATest : BTest()`.
Parent class (BTest) has some tests. ATest extends BTest and overrides something. As a result, it actually has to be run, it's just all the tests are located in BTest.
Also, `@Test` might be present in docs or comments.
I'm not sure there's an easy way to take into account all the cases and we might actually go on with the current approach. But just in case you have some ideas. Maybe there's some parser library that would actually parse all the tests?

Reply:
you are right, I'll think about it